opm/simulators/wells/WellHelpers.hpp

```diff
@@ -232,9 +232,21 @@ namespace Opm {
             return cube;
         }
 
+        // explicite transpose of  dense matrix due to compilation problems
```

Comment:
Care to mention what compilation problems you are talking about?

Reply:
Some kind of template issue.. (maybe the blockmatrix/fieldmatrix thing). I did not see way it did not manage to use the correct function.